cardano-node/src/Cardano/Node/Configuration/Logging.hs

```diff
@@ -211,3 +232,56 @@ createLoggingLayer ver nodeConfig' = do
 
 shutdownLoggingLayer :: LoggingLayer -> IO ()
 shutdownLoggingLayer = shutdown . llSwitchboard
+
+-- The node provides the basic node's information for TraceForwarderBK.
+-- It will be sent once TraceForwarderBK is connected to an external process
+-- (for example, RTView).
+nodeBasicInfo :: NodeConfiguration
+              -> Consensus.TopLevelConfig (CardanoBlock c)
+              -> UTCTime
+              -> IO [LogObject Text]
+nodeBasicInfo nc cfg nodeStartTime' = do
+  let CardanoLedgerConfig cfgByron cfgShelley cfgAllegra cfgMary = Consensus.configLedger cfg
+      protocolDependentItems =
+        case ncProtocol nc of
+          ByronProtocol ->
+             let genesis = byronLedgerConfig cfgByron
+             in [ ("systemStartTime",  show (getSystemStart $ Consensus.configBlock cfg))
+                , ("slotLengthByron",  show (fromByronSlotLength $ genesisSlotLength genesis))
+                , ("epochLengthByron", show (fromByronEpochSlots $ Gen.configEpochSlots genesis))
+                ]
+          ShelleyProtocol ->
+            let genesis = shelleyLedgerGenesis $ shelleyLedgerConfig cfgShelley
+            in [ ("systemStartTime",          show (SL.sgSystemStart genesis))
+               , ("slotLengthShelley",        show (mkSlotLength $ SL.sgSlotLength genesis))
+               , ("epochLengthShelley",       show (SL.sgEpochLength genesis))
+               , ("slotsPerKESPeriodShelley", show (SL.sgSlotsPerKESPeriod genesis))
+               ]
+          CardanoProtocol ->
+            let genesisShelley = shelleyLedgerGenesis $ shelleyLedgerConfig cfgShelley
+                genesisAllegra = shelleyLedgerGenesis $ shelleyLedgerConfig cfgAllegra
+                genesisMary    = shelleyLedgerGenesis $ shelleyLedgerConfig cfgMary
+            in [ ("systemStartTime",          show (SL.sgSystemStart genesisShelley))
```

Comment:
You are using `show` here, what does the output look like? Best to make sure it doesn't look like `SlotLength { getSlotLength :: ... }` or something, because that makes parsing brittler.